### PR TITLE
docs(): add instructions for mac apps

### PR DESCRIPTION
**Pull request by mhartington**
Closes https://github.com/jwilm/alacritty/issues/687

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,6 +176,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+To build an application for macOS, run
+
+```sh
+make app
+```
```
Comment (martinlindhe) [inline README.md]: `cp -r target/release/osx/Alacritty.app /Applications/Alacritty.app`

as outlined in https://github.com/jwilm/alacritty/wiki/MacOS-application-bundle

File: README.md [hunk 2]
```diff
@@ -176,6 +176,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+To build an application for macOS, run
+
```
Comment (martinlindhe) [inline README.md]: `cargo build --release`

Comment (jwilm) [inline README.md]: Should this be repeated here? It's mentioned just a few lines earlier.

Comment (mhartington) [inline README.md]: yeah, I didn't add this because it was mentioned earlier. This is a note similar to Desktop entry for Linux section

Comment (martinlindhe) [inline README.md]: true, ignore that :)

**Comment by mhartington** [comment]
Copy has been update from feedback.

**Comment by jwilm** [comment]
Thanks for the PR!

**Pull request merged**